Shopify/argo-checkout-template
# Fix yarn.lock and missing dependency

This PR fix yarn.lock and missing dependency

## How to test?
- Run `yarn && yarn generate --type=CHECKOUT_ARGO_EXTENSION`
- Verify that extension is generated as normal
